projects/storefrontlib/src/recipes/b2c-storefront.module.ts

```diff
@@ -28,7 +24,7 @@ import { HttpClientModule } from '@angular/common/http';
     }),
     provideDefaultConfig(layoutConfig),
     provideDefaultConfig(mediaConfig),
-    provideDefaultConfigFactory(defaultCmsContentConfig),
+    ...staticCmsStructureProviders,
```

Comment:
IMO `provideDefaultConfigFactory(defaultCmsContentConfig),` was much easier to reason about than `...staticCmsStructureProviders`. Can we change the name to make it a bit more self-explanatory?